src/components/LooMap/LooMap.tsx

```diff
@@ -313,7 +313,7 @@ const LooMap: React.FC<LooMapProps> = ({
           }
         >
           {alwaysShowGeolocateButton && <LocateMapControl />}
-          <Media greaterThan="md">
+          <Media greaterThanOrEqual="sm">
```

Comment:
Everything is looking good with regards to the zoom-out on geolocation feature.

These controls were never designed to be shown on mobile - here’s an example of what they look like on mine to illustrate:

![4CAE97B7-E3EE-49FE-AFD0-823C5F26C2D7](https://user-images.githubusercontent.com/1771189/188248060-ce8ba009-5e7d-4b02-9786-e750767db621.png)
![4904A53E-E13D-4040-967E-87EE1484DB06](https://user-images.githubusercontent.com/1771189/188248065-2a0d9612-d300-43d6-8aa4-4a2aa5729b55.png)

normally they look alright, but we shift them up on desktop away from the toilet description when it’s opened which - as shown by the screenshot - looks a bit off on mobile.

i suggest we leave this at `md` so it’s hidden on mobile still for now; then we can open an issue or another pr to investigate ways we can include the controls in a nicer way… or make a decision as to whether we’d like to include them at all.